.github/workflows/pull-request.yaml

```diff
@@ -85,14 +85,15 @@ jobs:
       CARGO_PROFILE_TEST_DEBUG: 0
     steps:
       - uses: actions/checkout@v3
-      - uses: Swatinem/rust-cache@v2
-      # Start the build process in the background. The following cargo test command will automatically
-      # wait for the build process to be done before proceeding.
-      - run: cargo build -p driver --tests &
-      - uses: yu-ichiro/spin-up-docker-compose-action@v1
+      - uses: engineerd/configurator@v0.0.5
```

Comment:
I think `foundry` has a Github action that we can use:

https://github.com/foundry-rs/foundry-toolchain